### DISCUSSION TITLE
Tagging opus files

### DISCUSSION THREAD

**Discussion by mgonzalezm** [complexlogic/rsgain#106] [Q&A]
Is it there a way to tag opus files using R128_*_GAIN tags and set the header gain to either R128_ALBUM_GAIN or R128_TRACK_GAIN but preserving the other one and setting it relative to the header gain ?

This is the standard way to do it, opusenc does this way when transcoding from FLAC files that have replaygain tags.

It seems to me that it is impossible to do this with rsgain because using either --opus-mode=t or 
--opus-mode=a, always clear the R128_*_GAIN tags

any help is appreciated.

**Comment by complexlogic** [comment]
Currently, this is not possible, but that is an interesting method.
